acceptance/bundle/deploy/registered_models/basic/script

```diff
@@ -0,0 +1,50 @@
+trace export NAME="my-registered-model-$UNIQUE_NAME"
+trace export COMMENT="original comment"
+trace export CATALOG_NAME="main"
+trace export SCHEMA_NAME="default"
+envsubst < databricks.yml.tmpl > databricks.yml
+
+title "create catalog and schema to test diff functionality"
+catalog_name="mycatalog-${UNIQUE_NAME}"
+schema_name="myschema-${UNIQUE_NAME}"
+trace $CLI catalogs create ${catalog_name} | jq '{full_name}'
+trace $CLI schemas create ${schema_name} ${catalog_name} | jq '{full_name}'
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+    trace $CLI schemas delete ${catalog_name}.${schema_name} --force
+    trace $CLI catalogs delete ${catalog_name} --force
+}
+trap cleanup EXIT
+
+deploy_registered_model() {
+    trace $CLI bundle plan
```

Comment:
recording "bundle debug plan" would be nice as well.

Reply:
I'll address this in a followup PR.